module5/labs/Week1/RefactoringPatterns.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: Refactoring Patterns Lab
+---
+
+## Disordered Orders
+
+Today, you are joining the Disordered Orders development team.  You have been tasked with refactoring their order-creating application.
+
+### Setup
+
+* Fork and Clone this [starter repo](https://github.com/turingschool-examples/DisorderedOrders)
+* Run `update-database` in the DisorderedOrdersMVC project
+* Run all test in the DisorderedOrdersTests project
+* If you run into any errors, reach out to your instructor!
+
+### The Refactors
+Implement the following refactoring patterns.  In your submission, we will ask for some reflection on each, so it would be helpful to keep notes as you go!
```

Comment:
Can we require that students check out a new branch for each refactor? So if it gets messy messy they can go easily to what it looked like before the refactor and start again?